### PR TITLE
alternate landing page second row with less hero images

### PR DESCRIPTION
**Pull request by sw-yx**
## What was changed:
second row now just uses bullet points

## Why?
too many heroes

before:

image

after:

image

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -185,40 +185,30 @@ export default function Home() {
...
-              styles.getStarted,
-            )}
-            to={useBaseUrl("/docs/concept-overview")}>
-            Learn more
-            </Link>
-          </div>
+          <ul>
+            <li><Link to={useBaseUrl("/docs/concept-overview")}> Overview </Link></li>
+            <ul>
+              <li><Link to={useBaseUrl("/docs/concept-workflows")}> Workflows </Link></li>
```
Comment (flossypurse) [inline src/pages/index.js:195]: I think these should all be in the same unordered list and change Overview to Introduction, i.e:
- Introduction
- Workflows
- Activities
- Workers
- More... etc

**Comment by flossypurse** [comment]
I do like this more, but I am planning on adding another section called "System design".
Would we be able to add third column there or would you want to see that added to one of the existing  two columns?

File: src/pages/index.js [hunk 2]
```diff
@@ -185,40 +185,30 @@ export default function Home() {
...
+              <li><Link to={useBaseUrl("/docs/concept-activities")}> Activities </Link></li>
+              <li><Link to={useBaseUrl("/docs/concept-workers")}> Workers </Link></li>
+            </ul>
+            <li>More: <Link to={useBaseUrl("/docs/concept-task-queues")}> Task Queues</Link>, <Link to={useBaseUrl("/docs/concept-signals")}> Signals</Link>, <Link to={useBaseUrl("/docs/concept-queries")}>Queries </Link></li>
```
Comment (sw-yx) [inline src/pages/index.js:199]:
```suggestion
            <li><Link to={useBaseUrl("/docs/concept-overview")}> Introduction </Link></li>
            <li><Link to={useBaseUrl("/docs/concept-workflows")}> Workflows </Link></li>
            <li><Link to={useBaseUrl("/docs/concept-activities")}> Activities </Link></li>
            <li><Link to={useBaseUrl("/docs/concept-workers")}> Workers </Link></li>
            <li>More: <Link to={useBaseUrl("/docs/concept-task-queues")}> Task Queues</Link>, <Link to={useBaseUrl("/docs/concept-signals")}> Signals</Link>, <Link to={useBaseUrl("/docs/concept-queries")}>Queries </Link></li>
```

**Comment by sw-yx** [comment]
re: third column or not - i think we'd have to play around with how it looks, it might get a bit busy, but we can cross that bridge when we get there

**Comment by flossypurse** [comment]
Awesome. Love it. Lets roll.

**Review by flossypurse — APPROVED** [review]

**Comment by flossypurse** [comment]
I don't know happened here - but somehow this got into a weird state and I can't figure out the merge conflict.
Creating a fresh PR for these changes here: https://github.com/temporalio/documentation/pull/377

**Pull request closed without merging**